main.gms

```diff
@@ -230,548 +217,1220 @@ $setGlobal c_description  REMIND run with default settings
 ***---------------------    51_internalizeDamages    ----------------------------
+*** (off):
+*** (DiceLikeItr): Internalize DICE-like damages (calculate the SCC) adjust cm_damages_SccHorizon. Requires cm_emiscen set to 9 for now.
+*** (BurkeLikeItr): Internalize Burke-like damages (calculate the SCC) adjust cm_damages_SccHorizo. Requires cm_emiscen set to 9 for now.
+*** (KWlikeItr): Internalize damage function based on Kalkuhl & Wenz (2020). Requires cm_emiscen set to 9 for now.
+*** (KWlikeItrCPnash): Internalize damage function based on Kalkuhl & Wenz (2020), but with Nash SCC, i.e. each region only internalizes its own damages. Requires cm_emiscen set to9 for now.
+*** (KWlikeItrCPreg): Internalize damage function based on Kalkuhl & Wenz (2020), but with regional SCC instead of a global uniform price. Requires cm_emiscen set to 9 for now.
+*** (KW_SEitr): Internalize damage function based on Kalkuhl & Wenz (2020) for upper limit based on standard error. Requires cm_emiscen set to 9 for now.
+*** (KWTCintItr): Internalize combined damages from Kalkuhl & Wenz (2020) and from tropical cyclones. Requires cm_emiscen set to 9 for now.
+*** (LabItr): Internalize labor supply damages based on Dasgupta et al. (2021). Requires cm_emiscen set to 9 for now.
+*** (TCitr): Internalize tropical cyclone damage function based on Krichene et al. (2022). Requires cm_emiscen set to 9 for now.
 $setGlobal internalizeDamages  off               !! def = off
 ***---------------------    70_water  -------------------------------------------
+*** (off): no water demand taken into account
+*** (exogenous): exogenous water demand is calculated based on data on water demand coefficients and cooling shares
+*** (heat): as exogenous only that vintage structure in combination with time dependent cooling shares as vintages and efficiency factors are also considered and demand is a function of excess heat as opposed to electricity output
 $setglobal water  off                 !! def = off
 ***---------------------    80_optimization    ----------------------------------
+*** (nash): Nash solution. Adjust cm_nash_autoconverge to needs.
+*** (negishi): calculates a Negishi solution (social planner)
+*** (testOneRegi):  solves the problem for one region for given prices (taken from gdx).
+***                ! Warning:  For development purposes only !
 $setGlobal optimization  nash         !! def = nash
 ***---------------------    81_codePerformance    -------------------------------
+*** (off): nothing happens
+*** (on):  test code performance: noumerous (30) succesive runs performed in a triangle, tax0, tax30, tax150, all growing exponentially,
+***                    therefore use carbonprice|exponential, c_emiscen|9, and cm_co2_tax_2020|0.
 $setGlobal codePerformance  off       !! def = off
 
 ***-----------------------------------------------------------------------------
-***                     SWITCHES and FLAGS
+***                     SWITCHES 
```

Comment:
The following still contains flags, so why this change? Also, adding an explanation of what a switch does and how it is different than what a flag does, might be useful

Reply:
flags are now starting with:
***-----------------------------------------------------------------------------
***                     FLAGS 
***-----------------------------------------------------------------------------